tests/test_sradb.py

```diff
@@ -0,0 +1,22 @@
+"""Tests for sradb.py
+"""
+
+import os
+import pytest
+from pysradb import SRAdb
+from pysradb.filter_attrs import guess_cell_type, guess_tissue_type, guess_strain_type
+from sqlite3 import OperationalError
+
+def test_not_valid_file():
+    """Test to check for error if file is either not
+        present or not a valid sqlite file"""
+    path = 'SRAmetadb.sqlite' 
+    try:
+        db = SRAdb(path)
+        assert False
```

Comment:
This is because while testing, `SRAmetadb.sqlite` file won't exist in the current directory, hence if it succeeds in establishing a connection, then it would be wrong, hence the assert False.

Reply:
Ummm, but an error would be raised before hand and hence this statement is practically unreachable?

Reply:
Yes, practically it shouldn't reach it.
This statement is just for edge case in case it makes a connection...

Reply:
I guess it should be removed then...

Reply:
Done, I've removed it in the latest commit. 
But travis is failing to build even though pytest passes all checks. It may be because of the black --check statement.